### PR TITLE
[WebAPI] TC dev for AudioParams nominal range for Chromium 52

### PR DESCRIPTION
**Pull request by renchenglei**
Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: Crosswalk for Android 22.52.557.0
Unit test result summary: pass 0, fail 1, block 0

https://crosswalk-project.org/jira/browse/CTS-1862

### CODE REVIEW

**Comment by renchenglei** [comment]
@Honry PTAL, Thanks a lot~

File: webapi/tct-webaudio-w3c-tests/webaudio/WebKit/COPYING [hunk 1]
```diff
@@ -0,0 +1,12 @@
+This test suite comes from
+https://chromium.googlesource.com/chromium/src.git
```
Comment (Honry) [inline webapi/tct-webaudio-w3c-tests/webaudio/WebKit/COPYING]: It's better directing to an accurate address.

**Comment by renchenglei** [comment]
@Honry I have modified the code, ptal, thanks!

**Comment by Honry** [comment]
LGTM

**Pull request merged**